Formula/hydra.rb

```diff
@@ -25,7 +25,20 @@ def install
       s.gsub! "/opt/local/lib", Formula["openssl@1.1"].opt_lib
       s.gsub! "/opt/local/*ssl", Formula["openssl@1.1"].opt_lib
       s.gsub! "/opt/*ssl/include", Formula["openssl@1.1"].opt_include
-      # Avoid opportunistic linking of subversion
+      # Avoid opportunistic linking of everything
+      s.gsub! "gtk+-2.0", "oh_no_you_dont"
```

Comment:
- Can we list these strings in an array `%w[gtk+-2.0 libgcrypt etc]` and loop over that?
- Let's not list things that are irrelevant to Homebrew, like `libafpclient libfbclient librfc` (and possibly others)

Reply:
Done now.

Note that although `libfreerdp2` isn't _strictly_ supplied by Homebrew yet I've left it in as you might have it if you do a `--HEAD` install on freerdp. 2.0 is in RC stage and will become stable before too long so `libfreerdp2` will eventually be on Homebrew bottles.